src/common/localization/en-us.ts

```diff
@@ -129,6 +129,11 @@ export const english: IAppStrings = {
         backEndNotAvailable: "Checkbox feature will work in future version of Form Recognizer service, please stay tuned.",
         addName: "Add a model name...",
         downloadJson: "Download JSON file",
+        errors: {
+            electron: {
+                cantAccessFiles: "Cannot access to files. Folder path '${folder}' or/and connection settings are invalid. Please check folder path and your connection settings."
```

Comment:
I don't think this conveys the correct message to the user. At this point the connection and project folder is already tested for existence. The issue is either with the label folder URI or the folder is not mounted correctly.
Note - training with on-prem is actually decoupled from project training set specified by connection and project folder. Thats why user must input label folder uri

Something like "Cannot access files in [label_folder_URI] for training. Please check if folder is mounted correctly